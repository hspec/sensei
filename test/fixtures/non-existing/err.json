{"version":"1.0","ghcVersion":"ghc-9.10.0","span":null,"severity":"Error","code":49196,"message":["Can't find test/fixtures/non-existing/Foo.hs"],"hints":[]}
