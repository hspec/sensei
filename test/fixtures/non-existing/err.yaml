version: '1.1'
ghcVersion: ghc-9.10.0
span: null
severity: Error
code: 49196
message:
- Can't find test/fixtures/non-existing/Foo.hs
hints: []
