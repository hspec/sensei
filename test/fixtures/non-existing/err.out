<no location info>: error: [GHC-49196]
    Can't find test/fixtures/non-existing/Foo.hs

